### PR TITLE
wishlist page done

### PR DESCRIPTION
**Pull request by TJain02**
Screenshot (161)

### CODE REVIEW

**Review by DevNirah7 — APPROVED** [review]
Looks good to commit!

File: CSS-Components/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    position: fixed;
+    right: 0;
+    left: 0;
+    top:0;
+    background-color:var(--dark-indigo);
+    padding:1rem;
+    color: var(--light-indigo);
+    z-index: 2;
+  }
+
```
Comment (Asypher00) [inline CSS-Components/wishlist.css:23]: avoid extra spacing, try beautify for proper indentation

File: CSS-Components/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    flex-grow: 0.5;
+    align-items: center;
+    gap: 0.5rem;
+  } 
+ 
+  .search-bar{
+    flex-grow: 3;
+    font-size: 1rem;
+  }
+
```
Comment (Asypher00) [inline CSS-Components/wishlist.css:35]: avoid extra spacing, try beautify for proper indentation

File: CSS-Components/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    width:12rem;
+    height: 17rem;
+}
+
+.wishlist-btn{
+    cursor: pointer;
+    width:0px;
+    height:0px;
+    position: absolute;
+    top: 0.1rem;
```
Comment (Asypher00) [inline CSS-Components/wishlist.css:80]: try using px for values below 10px

**Review by Asypher00 — COMMENTED** [review]
suggested some changes please check

**Review by shivamSspirit — COMMENTED** [review]
good to go

**Pull request merged**